meson.build

```
@@ -150,6 +149,7 @@ if not get_option('source-only')
     sdl_options += 'test=false'
     sdl_options += 'use_sensor=disabled'
     sdl_options += 'use_haptic=disabled'
+    sdl_options += 'use_hidapi=disabled'
     sdl_options += 'use_audio=disabled'
     sdl_options += 'use_cpuinfo=disabled'
     sdl_options += 'use_joystick=disabled'
```

To fix msys2 builds:
```suggestion
    sdl_options += 'use_joystick=disabled'
    sdl_options += 'use_joystick_xinput=disabled'
```

---

Thanks, let's keep that until they fix it in wrapdb.